deltachat-ios/Helper/NotificationManager.swift

```diff
@@ -55,7 +75,8 @@ public class NotificationManager {
                     if chat.isMuted {
                         return
                     }
-
+                    
+                    self?.registerNotificationBackgroundTask(msgId: messageId)
```

Comment:
i think, this registering may come too late.

we should register the task before it is really started, otherwise we may get 0xdead10cc eg. in the getChat() call a line above.